plasmapy/formulary/relativity.py

```diff
@@ -9,13 +9,12 @@
 import numpy as np
 
 from astropy import units as u
-from plasmapy import utils
 from astropy.constants import c
+from plasmapy import utils
+from plasmapy.utils.decorators import validate_quantities
```

Comment:
I'm wondering if the name `decorators` (which I think we've used for a while) is a bit general.  Perhaps change `plasmapy.utils.decorators` to `plasmapy.utils.validators`?

Reply:
Well, I see `plasmapy.utils.decorators` as a general grab-bag of decorators for PlasmaPy.  Within the `decorators` sub-package I've split the **"check"** decorators into `checks.py` and the **"validate"** decorators into `validators.py`.  I just also make the key classes and functions accessible in the `plasmapy.utils.decorators` name space.

I could import from `plasmapy.utils.decorators.validators`, but I think `validate_quantities` is fairly self explaining.

Reply:
Understood.  Makes sense!

Reply:
I agree that this makes sense as it stands. :+1: